src/tools/initCommandHelpers.ts

```diff
@@ -62,15 +62,23 @@ export const installReactNative = async (
   await handleOperation(projectName, OperationKey.Install, async () => {
     const templateVersion = options['templateVersion']
       ? `@${options['version']}`
-      : ''
-    await execa.command(
-      `npx react-native init ${projectName} --template react-native-template-typescript${templateVersion} --skip-install `
-    )
+      : undefined
+
+    if (templateVersion) {
+      await execa.command(
+        `npx react-native init ${projectName} --template react-native-template-typescript${templateVersion} --skip-install `
+      )
+    } else {
+      // It will install the latest RN version that already includes first-class TS support
```

Comment:
This is one of the few scenarios where I find reasonable to add comments.

[ref](https://github.com/ryanmcdermott/clean-code-javascript#only-comment-things-that-have-business-logic-complexity).